ChromecastSampleApp/ChromecastSampleApp/Players/PlayerViewController.m

```diff
@@ -87,7 +87,9 @@ - (void)viewDidLoad {
   // Init the castManager in the ooyalaPlayer
   [self.ooyalaPlayer initCastManager:self.castManager];
   [self.ooyalaPlayer setEmbedCode:self.embedCode];
-  [self.ooyalaPlayer play];
+  if( self.mediaInfo.startPlaying ) {
```

Comment:
I think we don't really need to involve the embed_code in this; that would be inferring what should be done vs. just explicitly asking the CastPlayer for what state it is in directly?